tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/remote/BallerinaRunRemoteDebugTest.java

```diff
@@ -86,6 +86,23 @@ public void testSuspendOnBallerinaJarRun() throws BallerinaTestException {
         clientLeecher.waitForText(20000);
     }
 
+    @Test(description = "Tests executable JAR debugging support with [--debug=<PORT>] option.")
+    public void testSuspendOnBallerinaJarRun2() throws BallerinaTestException {
+        String executablePath = Paths.get("target", "bin", testProjectName.replaceAll("-", "_") + ".jar")
+                .toFile().getPath();
+        LogLeecher clientLeecher = new LogLeecher(executablePath);
+        balClient.runMain("build", new String[]{}, null, new String[]{},
+                new LogLeecher[]{clientLeecher}, debugTestRunner.testProjectPath);
+        clientLeecher.waitForText(20000);
```

Comment:
This code segment is repeated in `testSuspendOnBallerinaJarRun1` and `testSuspendOnBallerinaJarRun2`. Shall we move this to a `utils` function?

Reply:
Added with ec57c71